homeassistant/components/blink/config_flow.py

```diff
@@ -61,6 +61,8 @@ async def async_step_user(
                 session=async_get_clientsession(self.hass),
             )
             await self.async_set_unique_id(user_input[CONF_USERNAME])
+            if self.source != SOURCE_REAUTH:
+                self._abort_if_unique_id_configured()
```

Comment:
I think reauth will also need to be adjusted, but I think that should be done in a follow-up PR